docs/client-development/cors.rst

```diff
@@ -0,0 +1,39 @@
+.. _client-development-cors:
+
+Cross-Origin Resource Sharing (CORS)
+====================================
+
+Some clients develop against Open Producten using single-page-application technology that
+runs completely in the browser, such as React, Angular or other frameworks.
+
+Open Producten must be deployed with an appropriate CORS-configuration for this.
+
+.. note:: We always recommend using an API gateway/own backend to communicate with Open
+   Zaak. It's simpler because you don't have to deal with CORS, and there's less risk
```

Comment:
```suggestion
.. note:: We always recommend using an API gateway/own backend to communicate with Open
   Producten. It's simpler because you don't have to deal with CORS, and there's less risk
```